packages/foreman/foreman/foreman.spec

```
@@ -1030,6 +1040,10 @@ exit 0
 %systemd_postun %{name}.socket
 
 %changelog
+* Wed Feb 24 2022 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl> - 3.0.1-2
```

Other pr is not showing linting error as it failed before that, I think fixing day should be enough,
```suggestion
* Thu Feb 24 2022 Ewoud Kohl van Wijngaarden <ewoud@kohlvanwijngaarden.nl> - 3.0.1-2
```